Dockerfile

```diff
@@ -9,13 +8,20 @@ RUN builder --config builder-config.yaml
 # use the official upstream image for the opampsupervisor
 FROM ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor AS opampsupervisor
 
-FROM alpine:latest
+FROM busybox:stable AS busybox
```

Comment:
Do we still need to use busybox to get the tools or could we remove these?

Reply:
removed